libs/design-system/error-pages/src/lib/error-pages.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { NotFoundPageComponent } from '@hra-ui/design-system/error-pages/not-found-page';
+
+/**
+ * All re-exported modules, components, directives, etc.
+ */
+const REEXPORTS = [NotFoundPageComponent];
+
+/**
+ * Error Pages Module - Contains components for displaying error pages
+ */
+@NgModule({
+  imports: REEXPORTS,
```

Comment:
Put `[NotFoundPageComponent]` in both the imports and exports instead of the constant. I think doing it that way will work better for tree shaking. I just haven't updated the other places to do so yet